styles/Navbar.module.css

```diff
@@ -212,13 +218,14 @@ navigation menu opens in mobile phones */
     color: rgb(12, 12, 12);
     text-shadow: none !important;
   }
+  
 
   .nav_active.active {
     color: white;
     text-shadow: none !important;
   }
 
   .navList_active {
-    background-color: rgb(62, 62, 252);
+    background-color: rgb(153, 120, 223);
```

Comment:
This is ok! you added bright colors

Reply:
@anmode definitely i will check

Reply:
that's cool! waiting for your next commit